src/app/GitUI/Editor/FileViewer.cs

```diff
@@ -425,34 +425,43 @@ public ArgumentString GetExtraDiffArguments(bool isRangeDiff = false, bool isCom
             };
         }
 
-        public ArgumentString GetDifftasticArguments(bool isRangeDiff = false)
+        public (ArgumentString args, string extraCacheKey) GetDifftasticArguments(bool isRangeDiff = false)
         {
             EnvironmentAbstraction env = new();
+            StringBuilder sb = new();
```

Comment:
nit
```suggestion
            StringBuilder extraCacheKey = new();
```

Reply:
extraCacheKeyBuilder (as extraCacheKey is a tuple member)